### ISSUE TITLE
List of tags feature with number of tagged entries

### ISSUE THREAD

**Issue by EnzoRondo** [xintrea/mytetra_dev#115]
You have interesting competitor: https://www.qownnotes.org/

these guys already made list of tags, it would be great to have same feature in mytetra, here is demo:

![image](https://user-images.githubusercontent.com/37320103/53935029-ce327580-409c-11e9-83b3-c8f2e7ddb044.png)

**Comment by john-cj** [comment]
Да, иметь единый список тегов, где-то сбоку или в виду виде отдельного окошка, хотя бы read-only, было бы здорово. Я такой опции не нашел.

Как я понимаю, автор предпочитает строгую иерархию, как в директориях. В интерфейсе программы это называется "ветками" и расположено в левом столбце. А тегам отведена какая-то крайне малозначительная роль.
